ci/jenkins/jobs/projects.yaml

```diff
@@ -399,7 +399,7 @@
           white_list: '{antrea_white_list}'
           only_trigger_phrase: true
           trigger_permit_all: false
-          trigger_phrase: ^(?!Thanks for your PR).*/skip-(e2e|all).*
+          trigger_phrase: ^(?!Thanks for your PR).*/skip-(e2e|all)($|\s).*
```

Comment:
Maybe add this change to all skip-... triggr_phase, too?

Reply:
I have added it to all skip trigger phrases in this file, are there other files need to update?